lib/id3c/api/routes.py

```diff
@@ -193,34 +193,49 @@ def get_identifier_set(name, *, session):
     return jsonify(set._asdict())
 
 
-@api_v1.route("/warehouse/identifier-sets/<name>", methods = ['PUT'])
+@api_v1.route("/warehouse/identifier-sets/<name>/<use>", methods = ['PUT'])
```

Comment:
I'd suggest not putting `use` into the URL here, as it's not part of the resources' PK and doesn't fit into the URL/resource hierarchy. Instead, `use` can be passed like `description` as a parameter. The param will obviously be required on creation, but could be allowed to be omitted for updates (for example, of just `description`). (The client can then choose if its ok with assuming pre-existence or not.)

Reply:
Ok, I agree it makes more sense to pass it as a form parameter. I'll update this shortly.

Reply:
This is close to working as intended, with the exception of allowing for updating `description` for an existing `identifier_set` without updating its `use`. Because `use` is a required field in the table, it's also required for any insert statement. 

I'm generating the sql statement dynamically now based on what form parameters are sent, so I'm not sure how to handle the case where only `description` is being updated. I started trying to switch to an update statement in those cases but that creates additional challenge of how to handle when `name` is not found (in which case it should attempt an insert and throw an error because `use` is not provided).

Reply:
UPDATE: commit f0b5ca162cabc9ec4d8e9b4f96dd6b0140298397 seems to be working the way it needs to. It required a bit more logic and code plus an additional select query but now handles all possible combinations of `use` and `description` form parameters.